test/e2e/memtierd.test-suite/damon.source.sh

```diff
@@ -1,67 +1,87 @@
-damon-idlepage-setup() {
-    if [ "$distro" != "debian-sid" ]; then
-        error "damon-idlepage-setup is implemented only for distro=debian-sid"
+#!/bin/bash
+# usage: kernel-download $version
+kernel-download() {
+    vm-command "[ -d linux ]" && vm-command "rm -rf linux"
+
+    vm-command "apt-get update" || error "failed to refresh apt package DB"
```

Comment:
Instead of running "apt-get update" on vm, let's use `distro-refresh-pkg-db`. Then this function will work on all distros that we support.